pkg/gokeyless.yaml

```diff
@@ -13,6 +13,7 @@ origin_ca_key:
 # Configure one or more private key directories.
 private_key_stores:
 - dir: /etc/keyless/keys
+- uri: pkcs11:token=SoftHSM2%20RSA%20Token;id=%03;slot-id=43989470?module-path=/usr/lib64/libsofthsm2.so&pin-value=1234
```

Comment:
This is the default config file that gets installed with the deb/rpm. Test data does not belong here.

Reply:
How do you propose to test using a sample SoftHSM key? I added this here since there are testing keys in the `keys` directory as well. Even if SoftHSM isn't install, this doesn't lead to a fatal error.

Reply:
Tests should have their own test config file. I don't know why those test keys were added before, but they don't belong either. Having this random test config installed when customers install the package makes no sense at all.